### DISCUSSION TITLE
Version number in launch screen

### DISCUSSION THREAD

**Discussion by VirtualRusty** [SafeExamBrowser/seb-win-refactoring#649] [Ideas]
We run university exams and to keep technical issues out of the exam room (as much as possible) we get students to log into the Inspera exam to the "enter exam password" prompt before entering the examination room. This way we can capture most SEB issues in the foyer and send them to a dedicated help desk. Often students download and/or launch the SEB as the first step, and stop there as they don't know what to do next. This situation offers staff the opportunity to check the SEB version is the right one for our installation - but there is no version number displayed on SEB Win like on SEB Mac - which shows the version number in the grab strip at the top of the window. 

Can we get the version number added to the launch screen somewhere? If not on the top grab bar then next to the title "Safe Exam Browser" so it reads something like "Safe Exam Browser 3.4.5"

This will assist in future releases where all students need the same version before entering the exam - we can just eyeball it and if its too old we can tell them to quit and upgrade their SEB at the same time.

**Comment by dbuechel** [comment]
In SEB for Windows (version 3.x), the version information is visible in the splash screens and the runtime as well as the about window (the latter of which can be opened via the action center `WIN+A` within SEB):

![grafik](https://github.com/SafeExamBrowser/seb-win-refactoring/assets/28347927/d50f7dca-4ba1-4d56-8966-7a95ee7bbda2)
![grafik](https://github.com/SafeExamBrowser/seb-win-refactoring/assets/28347927/70b215ed-2ebd-4d9c-8526-2eeb5cb06ad5)
![grafik](https://github.com/SafeExamBrowser/seb-win-refactoring/assets/28347927/b54be7f2-5b68-413c-95cb-13083479f718)

**Reply by VirtualRusty** [reply]
Thanks, I understand this, we check the splash screens at the door after the students launch SEB by the SSO login that launches Inspera, however, my original post relates to before that. Each exam session we have about 400 students file through the foyer with laptops in their hands and we walk around and around for about 40 mins checking what's on their screens as they log in to their laptops. If we can see that a prematurely launched SEB that is the wrong version we can deal with it straight away. Believe me, doing this routine twice a day for two weeks becomes exhausting, so anything that cuts down on lost time is a bonus. It could be implemented in the content area on both Win and Mac versions to standardise, then we can spot the issues in about 2 seconds.


![SEB splash copy](https://github.com/SafeExamBrowser/seb-win-refactoring/assets/120538228/98ffb40b-26a2-4bec-a264-05258923f28d)

**Reply by dbuechel** [reply]
@danschlet I guess that's then something we'd need to coordinate for all platforms?

**Reply by VirtualRusty** [reply]
Yeah that would be good. The text on that page is a bit confusing for SSO Enterprise exams where the .seb config loads automatically. We just have to tell the student to quit and go to the uni.inspera.com page. Don't know how you say that on the page but perhaps tell them to contact an exams staff member.
